### PR TITLE
Mobile skeleton milestone

### PR DESCRIPTION
**Pull request by luis-pomare**
Added toolbar and header of the mobile version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/bd800fb8b4.js" crossorigin="anonymous"></script>
+  <title>Luis Pomare</title>
+</head>
+<body>
+  <section class="toolbar">
+    <h2 class="welcome">Welcome</h2>
+    <img class="menu" src="images/menu.svg" alt="Menu">
```
Comment (Whoistolu) [inline index.html]: - [x] [REQUIRED] Kindly ensure that the hamburger icon has a `nav` tag.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @luis-pomare ,

Good job so far!
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

To Highlight:
✔️ Descriptive README file
✔️ Proper Gitflow

- [x] [REQUIRED] Kindly ensure that the first letter of your commit messages starts with a Capital letter. This makes your project more professional.
- [x] It will be nice to have a descriptive Pull Request in your project.  It should have a short summary describing introduced changes in more detail. Please [Read this](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for more understanding.

_Check the inline comments under the review for sections to improve._


Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by luis-pomare** [comment]
#what is new with this pull request
-Created index.html file with the structure of a mobile version of the web page. has created the two main sections of the mobile version( The toolbar and the header). There is only plane HTML and CSS not dynamic reactions with Javascript.
-Added styles with styles.css including one background image for the header and positioning with flexbox layout.
-A figma template has been used for the development of this first milestone and some of the images were exported from that template. Some other images (essentially the icons) were created and injected using awesome fonts web application.
-The microverse workflow and linters were applied locally and remotely.

**Review by Kingstalux — APPROVED** [review]
Hi @luis-pomare , 
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Good job on implementing the changes required 👍  Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏
- [ ] (OPTIONAL) You can make your README file more professional by adding a live demo link and information on how a user can set up your repo locally and run it on their machine.
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**